qemu/qemu.go

```diff
@@ -1689,3 +1709,17 @@ func LaunchCustomQemu(ctx context.Context, path string, params []string, fds []*
 	}
 	return errStr, err
 }
+
+// -pci devices don't play well with Z hence replace them with corresponding -ccw devices
+// See https://wiki.qemu.org/Documentation/Platforms/S390X
+func init() {
+	if runtime.GOARCH == "s390x" {
```

Comment:
I think this is a very bad idea. We should use the generic versions for all those that have aliases in QEMU. Those that don't should be managed explicitly.

Reply:
Actually the QEMU team is planning on deprecating those alias in favour of the -ccw alternatives. Hence this was done.

Reply:
@rbradford Keeping them to -ccw would also allow us to have better ABI compatibility

Reply:
@rbradford @devimc Do you still want me to make this change?